css/individual.css

```diff
@@ -533,7 +536,135 @@
         font-family: "Circe Regular";
         font-size: 12px;
     }
-    .individual__slider-container{
+
+    .individual__slider-container {
         display: flex;
     }
+}
+
+@media (min-width: 1080px) {
+    .individual__title {
+        font-size: 40px;
+    }
+
+    .individual__text {
+        font-size: 35px;
+    }
+
+    .individual__card-number-text {
+        font-size: 27px;
+    }
+
+    .individual__card-number {
+        font-size: 58px;
+    }
+
+    .individual__card {
+        height: 575px;
+    }
+
+    .individual__name-title {
+        font-size: 20px;
+    }
+
+    .individual__name-text {
+        font-size: 18px;
+        width: 280px;
+    }
+
+    .desing-img {
+        display: none;
+    }
+
+    .individual__card--last {
+        justify-content: space-around;
+    }
+
+    .individual__price {
+        font-size: 22px;
+    }
+
+    .individual__card--inner::after {
+        font-size: 18px;
+        top: 500px;
+        right: 120px;
+    }
+
+    .individual__name-circle {
+        font-size: 18px;
+        padding-left: 10px;
+    }
+
+    .individual__card--last::before {
+        background: url(../assets/img/individual-ill-lg.png) no-repeat;
+        top: 170px;
+        left: 15px;
+        width: 250px;
+        height: 300px;
+    }
+
+    .individual__slider-item {
+        width: 320px;
+        height: 430px;
+    }
+
+    .individual__slider-item--cherry {
+        background: url(../assets/img/berries-lg.png)
+    }
+
+    .individual__slider-item--design {
+        padding-top: 333px;
+    }
+
+    .individual__slider-item--masrhmello {
+        background: url(../assets/img/marsh-lg.png)
+    }
+
+    .individual__card--last {
+        justify-content: flex-end;
+        padding-right: 70px;
+    }
+
+    .individual__card--last .slider-button--left, .slider-button--right, .individual__slider-item {
+        margin-left: 40px;
+    }
+
+    .individual__card--last .slider-button {
```

Comment:
Can you use separate element instead of this selector

Reply:
done